esp32c6-hal/Cargo.toml

```diff
@@ -75,12 +75,15 @@ required-features = ["eh1"]
 
 [[example]]
 name              = "embassy_hello_world"
-required-features = ["async", "embassy"]
+required-features = ["embassy"]
 
 [[example]]
 name              = "embassy_wait"
-required-features = ["async", "embassy"]
+required-features = ["embassy", "async"]
+
+# [profile.dev]
+# opt-level = 1
```

Comment:
Please remove this.

Reply:
Can we also remove it from the C3 Cargo.toml, seems it was accidentally committed https://github.com/esp-rs/esp-hal/blame/main/esp32c3-hal/Cargo.toml#L86.

Reply:
Sure, let's do that in a different PR, though.